docs/tutorial/reduction.rst

```diff
@@ -0,0 +1,982 @@
+.. meta::
+  :description: The reduction tutorial on HIP
+  :keywords: AMD, ROCm, HIP, reduction, tutorial
+
+*************************************************************
+Tutorial: Reduction
+*************************************************************
+
+Reduction is an algorithmic operation that doesn't trivially map to GPUs but is
+ubiquitous enough to be of general interest while providing an excuse to
+introduce some key considerations of designing GPU algorithms and optimizing
+them.
+
+*This article is a rejuvenation and extension of the invaluable work of Mark
```

Comment:
I don't see a reason for this to be in italics. 
Also suggest you provide a link to Mark's work in this area.

Reply:
Fixed.